mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -61,7 +61,7 @@ static Value createLinalgBodyCalculationForElementwiseOp(
 
   if (isa<tosa::AbsOp>(op) && isa<IntegerType>(elementTy)) {
     auto zero = rewriter.create<arith::ConstantOp>(
-        loc, rewriter.getZeroAttr(elementTy));
+        loc, rewriter.getZeroAttr(converter.convertType(elementTy)));
```

Comment:
Is this the same as
```suggestion
        loc, rewriter.getZeroAttr(convertedElementTy));
```
? `args` are the type-converted operands (i.e. `op->getOperands()`), and we have  `auto convertedElementTy = args[0].getType();`  above.

Reply:
Ok. I did ask that when you were giving me the intro to this ticket and you said they weren't. I'll have a look at the documentation to confirm.

Reply:
I don't recall that, but I'm sorry if I had mislead you in any way.